spec/intl.html

```diff
@@ -1697,6 +1697,9 @@ <h1>
             Usually the [[Year]] field will contain the same value given by CalendarDateYear, but may contain the previous or next year if the week number in the [[Week]] field overlaps two different years.
             See also ToISOWeekOfYear.
           </p>
+          <p>
+            It returns *undefined* in [[Week]] and [[Year]] field for calendars that do not have a well-defined week calendar system.
```

Comment:
In the Temporal champions meeting we discussed that this should have a link to Intl. I've opened an [issue](https://github.com/tc39/proposal-intl-era-monthcode/issues/15) in the Intl Era/Month Codes proposal to suggest moving week numbering into scope, so that issue seems like a natural thing to link to. How about an editor's note that mentions this will be further specified there?